tests/test_save_load.py

```diff
@@ -227,6 +227,27 @@ def test_exclude_include_saved_params(tmp_path, model_class):
     os.remove(tmp_path / "test_save.zip")
 
 
+@pytest.mark.parametrize("model_class", [A2C, TD3])
+def test_save_load_env_cnn(tmp_path, model_class):
+    """
+    Test loading with an env that requires a CnnPolicy.
+    This is to test wrapping and observation space check.
+    We test one on-policy, one off-policy and HER
```

Comment:
"We test ... HER algorithm" but it is not included? Other than that checks out!

Reply:
oups, yes, I included it at first but then realized it requires a GoalEnv and that this wrapping is already tested in `test_her.py`.